.github/workflows/deploy-java-docs.yml

```diff
@@ -0,0 +1,36 @@
+    name: GitHub Pages Deploy Action
+    on:
+        # Branch name(s) which have the source file for site/documentation
+        push:
+            branches:
+                - "*"
+                #- "develop"
+        pull_request:
+            branches: 
+                - "*"
+    
+    jobs:
+        deploy-pages:
+            name: Deploy to GitHub Pages
+            runs-on: ubuntu-latest
+            steps:
+            - uses: actions/checkout@v2
+            - name: Set up JDK 12
+              uses: actions/setup-java@v1
+              with:
+                java-version: 12
+            - name: Grant execute permission for gradlew
+              run: chmod +x gradlew
+            #- name: Build and run tests
+            #  run: ./gradlew build
+            - name: Generate API documentation
+              if: ${{github.event_name == 'push'}}
```

Comment:
I think this line(27) and line number 30 both are unnecessary. You can remove lines 8 till 10 including 27 and 30 and it should work just the same.

Reply:
fixed, thank you 👍